docs/wip/exchange-onboarding.md

```diff
@@ -0,0 +1,352 @@
+    }'
+Warning: Those 12 mnemonic words given for the backup phrase act as an example. Do not use them on a production system. See the section below about mnemonic codes for more information.
+
+As a response, the API provides you with a wallet id used in subsequent requests to uniquely identity the wallet. Make sure to store it / write it down. Note that every API response is jsend-compliant; Cardano also augments responses with meta-data specific to pagination. More details in the section below about Pagination.
+
+    {
+        "status": "success",
+        "data": {
+            "id": "Ae2tdPwUPE...8V3AVTnqGZ",
+            "name": "MyFirstWallet",
+            "balance": 0
+        },
+        "meta": {
+            "pagination": {
+                "totalPages": 1,
+                "page": 1,
+                "perPage": 1,
+                "totalEntries": 1
+            }
+        }
+    }
+You have just created your first wallet. Information about this wallet can be retrieved using the GET /api/v1/wallets/{walletId} endpoint as follow:
+
+    curl -X GET https://localhost:8090/api/v1/wallets/{{walletId}} \
+         -H "Accept: application/json; charset=utf-8"              \
+         --cacert ./scripts/tls-files/ca.crt                       \
+
+## Receiving Money
+
+To receive money from other users you should provide your address. This address can be obtained from an account. Each wallet contains at least one account, you can think of account as a pocket inside of your wallet. Besides, you can view all existing accounts of a wallet by using the GET /api/v1/wallets/{{walletId}}/accounts endpoint as follow:
+
+    curl -X GET https://localhost:8090/api/v1/wallets/{{walletId}}/accounts?page=1&per_page=10 \
+         -H "Accept: application/json; charset=utf-8"                                          \
+         --cacert ./scripts/tls-files/ca.crt                                                   \
+Since you have, for now, only a single wallet, you’ll see something like this:
+
+    {
+        "status": "success",
+        "data": [
+            {
+                "index": 2147483648,
+                "addresses": [
+                    "DdzFFzCqrh...fXSru1pdFE"
+                ],
+                "amount": 0,
+                "name": "Initial account",
+                "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"
+            }
+        ],
+        "meta": {
+            "pagination": {
+                "totalPages": 1,
+                "page": 1,
+                "perPage": 10,
+                "totalEntries": 1
+            }
+        }
+    }
+
+Each account has at least one address, all listed under the addresses field. You can communicate one of these addresses to receive money on the associated account.
+
+## Sending Money
+
+In order to send money from one of your account to another address, you can create a new payment transaction using the POST /api/v1/transactions endpoint as follow:
```

Comment:
s/one of your account/one of your accounts/

Reply:
s/can/must/

s/as follow//

Reply:
s/money/Ada/

It is of course a form of money, but we might as well be precise, especially with exchanges as our audience.